static/src/javascripts/projects/common/modules/commercial/dfp/apply-creative-template.js

```diff
@@ -75,10 +75,8 @@ define([
         var creativeConfig = fetchCreativeConfig();
 
         if (creativeConfig) {
-            return Promise.all([
```

Comment:
When the consumer of the returned promise sees it resolve, what should they be able to presume about the state of the adSlot?